### DISCUSSION TITLE
Question: Does the app ALWAYS download all the media to the local folder?

### DISCUSSION THREAD

**Discussion by ux-git** [sircharlo/meeting-media-manager#2199] [Q&A] [answered]
I use automatic media sync feature all the time and it works really well. But I noticed that in the folder directory there's only one meeting, the one when I manually clicked the "Update media folders" button. 

So the question is: should I always manually click the button in order to download the files locally, or does the app automatically download them anyways on the launch sync?

**Comment by mtdvlpr** [comment, marked as answer]
Hi @ux-git, if you turned on the `Automatically initiate media sync` option in the settings, it should start downloading all media for that week on startup. If this doesn't happen, it could be a bug.

**Reply by ux-git** [reply]
Alright, thanks. I'll be testing this and going to send a bug report in case of it doesn't work as intended.

**Reply by mtdvlpr** [reply]
I will consider this as resolved for now :)
